decidim-accountability/app/commands/decidim/accountability/admin/create_status.rb

```diff
@@ -5,40 +5,12 @@ module Accountability
     module Admin
       # This command is executed when the user creates a Status from the admin
       # panel.
-      class CreateStatus < Decidim::Command
-        def initialize(form, user)
-          @form = form
-          @user = user
-        end
+      class CreateStatus < Decidim::Commands::CreateResource
+        fetch_form_attributes :key, :name, :description, :progress, :component
 
-        # Creates the status if valid.
-        #
-        # Broadcasts :ok if successful, :invalid otherwise.
-        def call
-          return broadcast(:invalid) if @form.invalid?
+        protected
```

Comment:
Same here about the `protected` method... Is this class inherited somewhere? 
```suggestion
        private
```

Reply:
Addressed in 29052b5d02